csaps/_sspumv.py

```diff
@@ -131,10 +149,16 @@ def __init__(self,
                  weights: Optional[UnivariateDataType] = None,
                  smooth: Optional[float] = None,
                  axis: int = -1,
-                 normalizedsmooth: bool = False):
+                 normalizedsmooth: bool = False,
+                 calculate_gcv: bool = False):
 
         x, y, w, shape, axis = self._prepare_data(xdata, ydata, weights, axis)
-        coeffs, smooth = self._make_spline(x, y, w, smooth, shape, normalizedsmooth)
+        if calculate_gcv:
+            coeffs, smooth, gcv = self._make_spline(x, y, w, smooth, shape, normalizedsmooth, calculate_gcv)
+            self.gcv = gcv
+        else:
+            coeffs, smooth = self._make_spline(x, y, w, smooth, shape, normalizedsmooth, calculate_gcv)
+            self.gcv = None
```

Comment:
Please move the condition to  `_make_spline` function and always return `gcv` (return None if calculate_gcv=False)